Formula/a/aspectj.rb

```diff
@@ -1,8 +1,8 @@
 class Aspectj < Formula
   desc "Aspect-oriented programming for Java"
   homepage "https://www.eclipse.org/aspectj/"
-  url "https://github.com/eclipse-aspectj/aspectj/releases/download/V1_9_21_1/aspectj-1.9.21.1.jar"
```

Comment:
Why change the url? https://github.com/eclipse/org.aspectj/ redirects to the initial one.

Reply:
oh, I was using the old url, did not notice it. reverting it now.